src/main/java/org/openstreetmap/atlas/geography/atlas/change/ChangeRelation.java

```diff
@@ -31,6 +31,10 @@
     private final Relation source;
     private final Relation override;
 
+    // Computing ChangeRelation members is very expensive, so we cache it here.
+    private transient RelationMemberList membersCache;
+    private transient Object membersCacheLock = new Object();
```

Comment:
never seen the transient keyword before! that's cool

Reply:
Yep, it means it will not be serialized, and when deserializing set to `null`.

Reply:
It has been the source of many bugs in the past :P